modeshape-jcr/src/main/java/org/modeshape/jcr/spi/index/provider/IndexChangeAdapters.java

```diff
@@ -393,14 +507,20 @@ protected Name convert( Path path ) {
         protected Name convertRoot( Path path ) {
             return Path.ROOT_NAME;
         }
+
+        @Override
+        protected void reorderNode( String workspaceName, NodeKey key, Name primaryType, Set<Name> mixinTypes, NodeKey parent,
+                                    Path newPath, Path oldPath, Path reorderedBeforePath ) {
+            // reordering should not really change the name...
```

Comment:
isn't the SNS a function of the path (segment) ? I thought that the node name (or mode:localName) does not reflect the SNS index. If it does however, then I'll have to change this,